modules/flight_interface/flight_interface_worker.py

```diff
@@ -33,6 +34,9 @@ def flight_interface_worker(
         print("ERROR: Worker failed to create class object")
         return
 
+    # Initalize queue with a maximum size of 1 to only hold latest odometry data
+    odometry_queue = queue_proxy_wrapper.QueueProxyWrapper(mp.Manager(), maxsize=1)
+
```

Comment:
Wait, why is this being created here? This queue should be passed in.

Add a check that the maximum size of the queue is 1 above line 32.